lib/analyzer/utils.ts

```diff
@@ -0,0 +1,24 @@
+import * as fs from "fs";
+import * as path from "path";
+
+/**
+ *  Remove directory recursively for all path passed as arg. It will stop removing
+ *  when it has removed everything or when the directory is not empty.
+ *
+ * @param {string[]} dirPath
+ */
+export function rmdirSyncRecursive(dirPath: string[]): void {
```

Comment:
lets not do that 😆  sounds bad.. what if the folder is `/var/tmp/blabla` - will we try to delete `var`?

Reply:
I cannot disagree with you! 😆 
The idea is clean up everything. But It doesn't make sense to remove the folder. I'm changing it.